### DISCUSSION TITLE
is this formula wrong here?

### DISCUSSION THREAD

**Discussion by clarkliming** [openpharma/mmrm#453] [General]
https://github.com/openpharma/mmrm/blob/b0741f8c6f25edfa799ce122c987dd2f7b820ece/simulations/missing-data-benchmarks/R/method/nlme-wrapper.R#L59

I belive we should use the  ~ visit|id as the covariance structure in nlme, what do you think? @danielinteractive

**Comment by danielinteractive** [comment]
Thanks @clarkliming , good question - from the doc of `corSymm` it says "Defaults to ~ 1, which corresponds to using the order of the observations in the data as a covariate, and no groups.", so if the order is correct, then it should be ok incidentally. But I guess to be more safe, `~ visit | id` is better.

**Reply by clarkliming** [reply]
~1|id would only work if

1. for unstructured covariance structure, the first m consecutive visits are collected
2. for toeplitz, consecutive visits are collected
3. for compound symmtry, does not have any impact

**Reply by danielinteractive** [reply]
ok thanks @clarkliming , then definitely better to change it, thanks again for finding this!
